### PR TITLE
Allow building with setuptools_scm 4 and 5

### PR DESCRIPTION
**Pull request by hroncok**
## Thanks for contributing a pull request!

If you are contributing for the first time or provide a trivial fix don't worry too
much about the checklist - we will help you get started.

## Contribution checklist:

(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)

- [x] wrote descriptive pull request text
- [ ] added/updated test(s) - not relevant
- [ ] updated/extended the documentation - not relevant
- [ ] added relevant [issue keyword](https://help.github.com/articles/closing-issues-using-keywords/) in message body - not relevant
- [x] added news fragment in [changelog folder](../tree/master/docs/changelog)
- [x] added yourself to `CONTRIBUTORS` (preserving alphabetical order)

### CODE REVIEW

**Review by gaborbernat — APPROVED** [review]

**Comment by gaborbernat** [comment]
Released now via https://tox.readthedocs.io/en/latest/changelog.html#v3-21-1-2021-01-13

File: docs/changelog/1799.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Enable building tox with ``setuptools_scm`` 4 and 5 by :user:hroncok
```
Comment (webknjaz) [inline docs/changelog/1799.misc.rst:1]: @hroncok you missed the backticks here ^

Comment (hroncok) [reply docs/changelog/1799.misc.rst:1]: Where?

Comment (gaborbernat) [reply docs/changelog/1799.misc.rst:1]: At your user name 👍🏻

Comment (hroncok) [reply docs/changelog/1799.misc.rst:1]: Oh, my. Sorry. https://github.com/tox-dev/tox/pull/1851

**Pull request merged**